utils/unittest/fixtures.go

```diff
@@ -197,7 +201,9 @@ func PayloadFixture(options ...func(*flow.Payload)) flow.Payload {
 func WithAllTheFixins(payload *flow.Payload) {
 	payload.Seals = Seal.Fixtures(3)
 	payload.Guarantees = CollectionGuaranteesFixture(4)
-	payload.Receipts = []*flow.ExecutionReceipt{ExecutionReceiptFixture()}
+	receipt := ExecutionReceiptFixture()
+	payload.Receipts = []*flow.ExecutionReceiptMeta{receipt.Meta()}
+	payload.Results = []*flow.ExecutionResult{&receipt.ExecutionResult}
```

Comment:
Maybe use a larger number of receipts and results here? Testing whether order is preserved with a single element is not very useful.

Reply:
now using 10 receipts